static/styles/subscriptions.css

```diff
@@ -466,13 +468,14 @@ h4.stream_setting_subsection_title {
 .user-groups-list {
     /*
         This height rule is temporary for this preparatory commit and
-        will be removed as we add other search/filter widgets in groups list
+        will be removed as we add other components widgets in groups list
         and its styling similar to streams list.
     */
-    height: 100% !important;
+    height: calc(100% - 42px) !important;
```

Comment:
So I probably should have brought this up before merging, but didn't notice it in time. 

This appears to be a change to the "stream name" search as well. Is it fixing a bug in that context, or introducing one?

Reply:
This rule would have no effect on stream settings overlay. It is applied here because stream settings have an extra row of widgets for (`subscribed | all`  `+`) . So the rule for height defined on [subscription.css #L469-L476](https://github.com/zulip/zulip/blob/main/static/styles/subscriptions.css#L469-L476) works well for stream list on stream settings overlay. I just tweaked the same rule for user groups as we have just one widget (search) row. I have marked it as temporary rule in the comments because it would be removed and we would be able to use height rule defined in [subscription.css #L469-L476](https://github.com/zulip/zulip/blob/main/static/styles/subscriptions.css#L469-L476) once  we have similar widget row for user group settings as well.